image-builder/README.md

```diff
@@ -1,8 +1,40 @@
-Image-builder on Scaleway
-========================
+# Official Image Builder on Scaleway
+
+[![Run on Scaleway](https://img.shields.io/badge/Scaleway-run-69b4ff.svg)](http://cloud.scaleway.com/#/servers/new?image=49eb4659-44a2-4d9c-bcc4-142185379e6e)
+
+Scripts to build the official Image Builder on Scaleway
+
+<img src="http://s27.postimg.org/dkmhiqu0j/builder.png" />
```

Comment:
If you don't add any html attributes like width/height, you can use the markdown syntax for image: `![](URL)`